### DISCUSSION TITLE
Thème msi

### DISCUSSION THREAD

**Discussion by Nonaud** [mathoudebine/turing-smart-screen-python#465] [Themes]
Hello As I didn't understand how to do it with python I used the software supplied with the 3.5" Turing "usb monitor" screen to create themes. So here is the preview and the background as well as the ".data" file
![PREVIEW MSI  ORANGE](https://github.com/mathoudebine/turing-smart-screen-python/assets/7085540/e096b6d5-97b9-4bd4-8106-fdd8ca3c2f93)

![PREVIEW MSI BLUE](https://github.com/mathoudebine/turing-smart-screen-python/assets/7085540/88344230-435f-458a-9380-3ba5284f21d1)
PS the fans do not exceed 1200rpm the preview seems to have a display bug which does not actually exist
[THEME MSI.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/14158765/THEME.MSI.zip)

**Comment by itsprstar** [comment]
Niceee!!
